src/main/steps/common/task-list/components/notification-banner/config/da_applicant.ts

```diff
@@ -17,4 +19,16 @@ export const DA_APPLICANT: NotificationBannerProps[] = [
       return caseData?.state === State.CASE_CLOSED;
     },
   },
+  {
+    ...notificationBanner[BannerNotification.SOA_SERVED_DA],
+    show: (caseData: Partial<CaseWithId>, userDetails: UserDetails): boolean => {
+      return (
+        caseData?.state !== State.CASE_CLOSED &&
+        isCaseLinked(caseData, userDetails) &&
+        (caseData.applicantsFL401
```

Comment:
can be written simpler with conditional operator.
caseData?.applicantsFL401?.response?.citizenFlags?.isApplicationToBeServed === YesOrNo.YES